data_model/clusters/ElectricalEnergyMeasurement.xml

```diff
@@ -67,50 +67,21 @@ Davis, CA 95616, USA
     <feature bit="1" code="EXPE" name="ExportedEnergy" summary="Measurement of energy provided by the server">
       <optionalConform choice="a" more="true"/>
     </feature>
-    <feature bit="2" code="CUME" name="CumulativeEnergy" summary="Measurements are accumulative">
-      <optionalConform choice="b" more="true">
-        <notTerm>
-          <feature name="EPHE"/>
-        </notTerm>
-      </optionalConform>
+    <feature bit="2" code="CUME" name="CumulativeEnergy" summary="Measurements are cumulative">
+      <optionalConform choice="b" more="true"/>
     </feature>
     <feature bit="3" code="PERE" name="PeriodicEnergy" summary="Measurements are periodic">
-      <optionalConform choice="b" more="true">
-        <notTerm>
-          <feature name="EPHE"/>
-        </notTerm>
-      </optionalConform>
-    </feature>
-    <feature bit="4" code="EPHE" name="EphemeralEnergy" summary="Measurements are ephemeral">
-      <mandatoryConform>
-        <andTerm>
-          <notTerm>
-            <feature name="CUME"/>
-          </notTerm>
-          <notTerm>
-            <feature name="PERE"/>
-          </notTerm>
-        </andTerm>
-      </mandatoryConform>
+      <optionalConform choice="b" more="true"/>
     </feature>
   </features>
+  <dataTypes/>
   <attributes>
-    <attribute id="0x0000" name="Measured" type="bool" default="false">
+    <attribute id="0x0000" name="Accuracy" type="ref_MeasurementAccuracyStruct">
```

Comment:
It properly does not matter right now, but the type here seems off 🤔

Reply:
it's somewhat correct - the reference is right, and leaving the reference in the scraped XML means we don't totally bork the xml->adoc path. The idea is to post-process these from the html <a> refs afterwards, because you're not guaranteed to have the ref_ in the same adoc file.

Reply:
We don't use the type currently for anything in the tests.

Reply:
Fair, it just looked a bit strange so I figured it might be a "bug" and I was also of the impression that we do not use this information yet, to is it not critical at all. Resolving this conversation 😄